### PR TITLE
Replace TestedCell source and remove ExerciseCell

### PR DESCRIPTION
**Pull request by Mraiih**
- [x] add tested_cell from BrooklinJazz/tested_cell
- [x] removed `makeup` and `makeup_elixir`  dependencies
- [x] removed `ExerciseCell` from MadLibs livebook
- [x] removed trace of `ExerciseCell`: exercise.ex, application.ex, main.css, main.js, string_concatenation_madlib.ex, tests
- [x] removed trace of `TestedCell`: tested_cell.ex, application.ex, main.css, main.js

### CODE REVIEW

File: exercises/boolean_operators.livemd [hunk 1]
```diff
@@ -67,11 +64,9 @@ b[____] --> or
 or --> true
 ```
 
-<!-- livebook:{"attrs":{"assertions":"assert answer == true","attempt":1,"code":"answer = nil","hint":"answer = true","hint_html":"<pre class=\"highlight\"><code><span class=\"n\">answer</span><span class=\"w\"> </span><span class=\"o\">==</span><span class=\"w\"> </span><span class=\"no\">true</span></code></pre>"},"kind":"Elixir.Utils.SmartCell.TestedCell","livebook_object":"smart_cell"} -->
+<!-- livebook:{"attrs":{"assertions":"assert answer == true","code":"answer = nil","solution":""},"kind":"Elixir.TestedCell","livebook_object":"smart_cell"} -->
```
Comment (BrooklinJazz) [inline exercises/boolean_operators.livemd:67]: I looks like this removed the hints. Can you put them back in?

File: exercises/fun_formulas.livemd [hunk 2]
```diff
@@ -43,11 +42,9 @@ In the Elixir cell below, calculate the percentage given `completed_items` and `
 
 Replace `nil` with your answer.
 
-<!-- livebook:{"attrs":{"assertions":"assert total_items >= completed_items, \"total_items should always be more than completed items, please reset the exercise\"\nassert percentage <= 100, \"percentage should be a float between 0 and 100. Please check the formula entered.\"\nassert percentage == completed_items / total_items * 100","attempt":0,"code":"completed_items = 10\r\ntotal_items = 100\r\n\r\npercentage = nil","hint":"percentage = completed_items / total_items * 100","hint_html":"<pre class=\"highlight\"><code></code></pre>"},"kind":"Elixir.Utils.SmartCell.TestedCell","livebook_object":"smart_cell"} -->
+<!-- livebook:{"attrs":{"assertions":"assert total_items >= completed_items,\n  \"total_items should always be more than completed items, please reset the exercise\"\n\nassert percentage <= 100,\n  \"percentage should be a float between 0 and 100. Please check the formula entered.\"\n\nassert percentage == completed_items / total_items * 100","code":"completed_items = 10\r\ntotal_items = 100\r\n\r\npercentage = nil","solution":""},"kind":"Elixir.TestedCell","livebook_object":"smart_cell"} -->
```
Comment (BrooklinJazz) [inline exercises/fun_formulas.livemd:45]: Can you add the solution (hint) here as well

**Review by BrooklinJazz — CHANGES_REQUESTED** [review]
Great work! The only issue is the solutions are missed from the `TestedCell`s. Once that's fixed it will be ready to merge.

**Comment by Mraiih** [comment]
Done, I've added back all forgotten solutions

**Comment by BrooklinJazz** [comment]
Awesome!

**Review by BrooklinJazz — APPROVED** [review]
Wahoo!

**Pull request merged**